docs/modeling/files.md

```diff
@@ -0,0 +1,24 @@
+---
+Title: Files
+---
+
+# Files
+Files can be uploaded and used within a project. They are stored in a persistent volume that is mounted at deployment time.
```

Comment:
Files are stored in PostgreSQL during the modelling stage. Once the application is deployed, the deployment-service creates a volume, and mounts that volume into all application connectors. A volume initialisation job copies the files to the volume so that all connectors can access them.

Reply:
Thanks for the additional info - I have included this in the description.